dispatch/tests/test___init__.py

```diff
@@ -0,0 +1,8 @@
+import dispatch.__init__
+from django.test import TestCase
+
+class __init__Test(TestCase):
+
+    def test_get_version(self):
+
+        return dispatch.__init__.__version__
```

Comment:
You don't need to include `__init__` here -- Python will automatically resolve the version like so:

```py
dispatch.__version__
```